xbmc/dbwrappers/mysqldataset.cpp

```diff
@@ -158,9 +168,6 @@ int MysqlDatabase::connect(bool create_new) {
       }
       else if (create_new)
       {
-        char sqlcmd[512];
-        int ret;
```

Comment:
Wouldn't we need to reset ret here if it was already used above? Just to be safe?